packages/ui/src/components/atoms/Dropdown.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+import { Image } from './Image';
+import { Input } from './Input';
+
+import {
+  cypherockRedIcon,
+  searchIcon,
+  triangleInverseIcon,
+} from '../../assets';
+import { LeanBoxProps, TempListItem } from '../molecules';
+
+const List = styled.ul`
+  position: absolute;
+  top: 100%;
+  right: 0;
+  width: 100%;
+  /* min-width: 420px; */
+  /* height: 192px; */
+  /* padding: 16px 0; */
+  list-style: none;
+  background-color: #2c2520;
+  border-radius: 8px;
+  box-shadow: 4px 4px 32px 4px #0f0d0b;
+  max-height: 200px;
+  overflow-x: hidden;
+  overflow-y: auto;
+  z-index: 10;
+  &:hover {
+    background-color: #272320;
+  }
+`;
+
+const ListItem = styled.li`
+  background-color: #46403c;
+  &:hover {
+    background-color: #272320;
+  }
+  /* margin-left: 40px; */
+`;
+
+const Container = styled.div<{ isOpen: boolean }>`
+  position: relative;
+  width: 100%;
+  border-radius: 8px;
+  background-color: #272320;
+  input {
+    padding-right: 30px;
+    border-color: ${props => (props.isOpen ? '#D4AF37' : '#272320')};
+  }
+  &:hover {
+    input {
+      border-color: #d4af37;
+    }
+    ${List} {
+      background-color: #191715;
+      cursor: pointer;
+    }
+  }
+`;
+
+const IconContainer = styled.div`
+  position: absolute;
+  right: 24px;
+  top: 60%;
+  transform: translateY(-50%);
+`;
+
+interface DropdownProps {
+  items: LeanBoxProps[];
+  shouldChangeColor?: boolean;
+}
+
+export const Dropdown: React.FC<DropdownProps> = ({
+  items,
+  shouldChangeColor,
+}) => {
+  const [search, setSearch] = useState('');
```

Comment:
How is this Dropdown communicating the selected value to it's parent?

Reply:
added